build/make-brew-formula.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Update the cockroach homebrew formula. Requires
+# github.com/samertm/homebrew-go-resources. Expected usage:
+#
+# 1. Tag a new release on https://github.com/cockroachdb/cockroach/releases
+#   a. The tag should have a name like v0.1-alpha
+#   b. This script currently requires the tag to start with a "v".
+# 2. `git pull` to grab the newly create git tag
+# 3. `git checkout <tag>`
```

Comment:
Ok. I've added a note to run an explicit `glock sync` which is mildly better than `make .bootstrap`.